components/error.tsx

```diff
@@ -1,11 +1,23 @@
 import { H3, Panel } from '@bigcommerce/big-design';
-import { ErrorMessageProps } from '../types';
+import { ErrorMessageProps, ErrorProps } from '../types';
 
-const ErrorMessage = ({ error }: ErrorMessageProps) => (
-    <Panel>
+const ErrorContent = ({ message }: Pick<ErrorProps, 'message'>) => (
+    <>
         <H3>Failed to load</H3>
-        {error && error.message}
-    </Panel>
-);
+        {message}
+    </>
+)
+
+const ErrorMessage = ({ error, renderPanel = true }: ErrorMessageProps) => {
+    if (renderPanel) {
```

Comment:
I sort of don't see the point of this boolean if we could just export the `ErrorContent` component above 🤔

Reply:
yeah, true... if we refactor, we could probably change `ErrorContent` to `ErrorMessage` and then `ErrorMessage` becomes `WrappedErrorMessage` or something

Reply:
Let's tackle it in another PR. It'll touch a lot more points than this PR intended on touching.

Reply:
ah yeah, I didn't take your original comment as something that needed to be done for this PR - more so if we did go down that route, we might want to update the component names